### PR TITLE
`osx-arm` notes

### PR DESCRIPTION
**Pull request by garrettwrong**
Minor updates to the README and install docs to reflect the current situation with the `PyFFTW` situation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,11 @@ install `aspire` safely in that environment.
 
 If you are unfamiliar with `conda`, the
 [Miniconda](https://docs.conda.io/en/latest/miniconda.html)
-distribution is recommended.
+distribution for `x86_64` is recommended.  For Apple silicon to use
+the `osx-arm` platform will currently require patching and building
+some dependencies from source.  The Intel `osx-64` install is still
```
Comment (j-c-c) [inline README.md]: This sentence needs rewording: "For Apple silicon to use
the `osx-arm` platform, patching and building
some dependencies from source is currently required."

File: docs/source/installation.rst [hunk 2]
```diff
@@ -34,6 +34,12 @@ to view Conda's installation instructions.
 .. note:: If you're not sure which distribution is right for you, go
    with `Miniconda <https://conda.io/miniconda.html>`__
 
+.. note:: For Apple silicon to use the `osx-arm` platform will
+   currently require patching and building some dependencies from
+   source.  The Intel `osx-64` install is still preferred even for
```
Comment (j-c-c) [inline docs/source/installation.rst]: Same as above.

**Review by j-c-c — CHANGES_REQUESTED** [review]
Just one suggestion.

File: README.md [hunk 3]
```diff
@@ -37,7 +37,11 @@ install `aspire` safely in that environment.
 
 If you are unfamiliar with `conda`, the
 [Miniconda](https://docs.conda.io/en/latest/miniconda.html)
-distribution is recommended.
+distribution for `x86_64` is recommended.  For Apple silicon to use
```
Comment (krissowat) [inline README.md:40]: This sentence sounds just a little funny.

**Review by krissowat — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (garrettwrong) [reply README.md:40]: Sure, I will update it since you guys didn't like it so much.

File: README.md (hunk 1 above)
Comment (garrettwrong) [reply README.md]: updated

**Review by krissowat — APPROVED** [review]
Sounds good.

**Review by j-c-c — APPROVED** [review]

**Review by janden — APPROVED** [review]

**Pull request merged**